doc/level1.rst

```diff
@@ -677,6 +677,11 @@ each of which:
   referenced by ``addr``. See also :ref:`preliminary recommendation`
   for how this key is selected.
 
+- If a key has multiple user ids, only one SHOULD be contained in
+  ``keydata``.  If present, that user id SHOULD match the ``addr``
+  attribute.  This is only relevant for keys which came from or were
```

Comment:
thanks. the "If present" confused me a little as i thought "why would a user id not be present?". 

Merging the first two sentences like this would unconfuse me and be shorter. 

"If a key has multiple user ids, only one SHOULD be contained in
 ``keydata`` and it SHOULD match the ``addr`` attribute. This is only ..."

Reply:
I see how that was confusing. I'm not too fond of this type of "SHOULD" in your suggestion though: It states a requirement that might not be possible to fulfill if a matching uid just isn't present.

I made another attempt